docs/07-theme-developer-tutorial/01-introduction.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Theme Developer Tutorial: 1. Introduction"
+short_title: "1 - Introduction"
+id: theme-developer-tutorial-intro
+---
+
+This tutorial will teach you how to create a Discourse Theme or Theme Component from the ground up. While this topic assumes no previous experience working on Discourse themes, it does assume some prior experience using [HTML](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML), [CSS](https://developer.mozilla.org/en-US/docs/Learn/CSS) and [JavaScript](https://developer.mozilla.org/en-US/docs/Learn/JavaScript). It'll also help if you [know your way around GitHub](https://guides.github.com/activities/hello-world/).
+
+## What are Discourse themes?
+
+> A theme or theme component is a set of files packaged together designed to either modify Discourse visually or to add new features.
+
+### Themes
+
+In general, themes are not supposed to be compatible with each other because they are essentially different standalone designs. You can have multiple themes installed, but you can't use two of them at the same time.
+
+### Theme Components
+
+Theme components are geared towards customising one aspect of Discourse. Because of their narrowed focus, theme components are almost always compatible with each other. This means that you can have multiple theme components running at the same time under any theme. You can think of theme components like apps on your phone.
```

Comment:
> You can think of theme components like apps on your phone.

If we want to do this analogy here, maybe we should find one for themes too.

Reply:
I just dropped it. Not really necessary.